pkg/operator/controllers/machinehealthcheck/doc.go

```diff
@@ -0,0 +1,27 @@
+package machinehealthcheck
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+/*
+
+The controller in this package aims to ensure that MachineHealthCheck objects
+exist and are correctly configured to automicatically mitigate non ready worker nodes.
+
+There are two flags which control the operations perfromed by the controller:
+
+aro.machinehealthcheck.enabled:
+- When set to false, the controller will noop and not perform any further action
+- When set to true, the controller continues on to check the managed flag
+
+aro.machinehealthcheck.managed
+- When set to false, the controller will attempt to remove the aro-machinehealthcheck CR from the cluster.
+  This should effectivly disable the MHC we deploy and prevent the automatic reconsilation of nodes.
```

Comment:
```suggestion
  This should effectively disable the MHC we deploy and prevent the automatic reconciliation of nodes.
```

Reply:
fixed, thank you! I've added a spell checker to VSCode, hopefully less typos appear in the future 😅